substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/FileEntry.java

```diff
@@ -59,4 +59,14 @@ public String getFullName() {
     public DirEntry getDirEntry() {
         return dirEntry;
     }
+
```

Comment:
Is there a reason why you want a toString for FileEntry but not for other elements of the debug info model?

Reply:
Originally, I had them everywhere; it saved much time while debugging. There's one on Range, too. I'm happy to get rid of them or add a couple more.  I think some were lost when we refactored these classes to be shared.

Reply:
Well, if hey are not particularly important then I would say less code is better

Reply:
Gone.